core/src/test/scala/org/apache/spark/sql/delta/DeltaColumnMappingTestUtils.scala

```diff
@@ -350,7 +350,7 @@ trait DeltaColumnMappingTestUtilsBase extends SharedSparkSession {
    * @param tableOrPath String
    */
   protected def convertToDelta(tableOrPath: String): Unit = {
-    sql(s"CONVERT TO DELTA $tableOrPath")
+    sql(s"CONVERT TO DELTA $tableOrPath NO STATISTICS")
```

Comment:
why this change?

Reply:
Since eventually we convert to delta with stats on a single commit, there is no need for that to make the tests pass. Removed.